src/votekit/ballot_generator.py

```diff
@@ -1993,68 +2042,106 @@ def generate_profile(
         )
 
 
-class ClusteredSpatial(BallotGenerator):
+class Clustered_DSpatial(BallotGenerator):
     """
-    Clustered spatial model for ballot generation.
-    Assumes the candidates are uniformly distributed in d dimensional space.
-    Voters are then distributed normally around them, and vote with
-    ballots based on Euclidean distance to the candidates.
+    Clustered spatial model for ballot generation. In some metric space
+    determined by an input distance function, randomly sample
+    each candidate's positions from input candidate distribution. Then
+    sample voters's positions from normal distributions centered around each
```

Comment:
Does it have to be normal?

Reply:
So it doesn't have to be normal, but it would have to be a distribution where the mean is a parameter. I kept it as normal because I thought that would often be the most natural choice. But I can definitely try add a more general form with some sort of check to make sure the input has a mean parameter.